qutip/tests/test_mesolve.py

```diff
@@ -920,5 +920,41 @@ def f(t, args):
             msg="evolution with feedback not proceding as expected")
 
 
+def test_non_hermitian_dm():
+    """Test that mesolve works correctly for density matrices that are
+    not Hermitian.
+    See Issue #1460
+    """
+    N = 2
+    a = destroy(N)
+    n = num(N)
+    x = (a + a.dag())/np.sqrt(2)
+    H = a.dag() * a
+
+    rho0 = x*fock_dm(N, 0)
+    tlist = np.linspace(0, 0.1, 2)
+
+    options = Options()
+    options.store_final_state = True
+    options.store_states = True
+    result = mesolve(H, rho0, tlist, e_ops=[x], options=options)
+
+    imag_part = np.abs(np.imag(result.expect[0][-1]))
+    # Since we used an initial state that is not Hermitian, the expectation of
+    # x must be imaginary for t>0.
+    msg = 'Mesolve is not working properly with a non Hermitian density ' +
```

Comment:
looks like the tests are failing cos of this linebreak, I guess wrapping in parentheses should fix it?

Reply:
Yes, indeed. It should be fixed in the last commit.